src/main/java/competition/subsystems/vision/VisionSubsystem.java

```diff
@@ -173,15 +173,20 @@ public boolean isEstimatedPoseReliable(EstimatedRobotPose estimatedPose, Pose2d
         var allTagIds = Arrays.asList(estimatedPose.targetsUsed.stream()
                 .map(target -> target.getFiducialId()).toArray(Integer[]::new));
         if (allTagIds.stream().anyMatch(id -> id < 1 || id > 8)) {
-            this.log.warn("Ignoring vision pose with invalid tag id. Visible tags: "
+            log.warn("Ignoring vision pose with invalid tag id. Visible tags: "
                     + String.join(", ", allTagIds.stream().mapToInt(id -> id).mapToObj(id -> Integer.toString(id)).toArray(String[]::new)));
             return false;
         }
 
         double distance = previousEstimatedPose.getTranslation().getDistance(estimatedPose.estimatedPose.toPose2d().getTranslation());
-        if(distance > errorThreshold.get()){
+        if(distance > errorThreshold.get()) {
+            log.warn(String.format("Ignoring vision pose because distance is %f from our previous pose. Current pose: %s, vision pose: %s.",
```

Comment:
I think we want a latch here or it will log every cycle once it's out of sync right?

Reply:
Right now the distance is 200m. I'm not fully convinced we're hitting this at all unless the properties in the robot are overriding the default.

Reply:
Also we have no way to reset the latch and make it log again?

Reply:
It would reset if the vision ever was reliable again and started being used?

Also I thought the units were inches, agree 200m is crazy. I had pitched that we try logging the delta and see what's normal

Reply:
I mistakenly merged this, but I added the same log throttling as the other vision code. The short story is that the logging latch only works for static log messages, as far as I can tell.